Source/WebCore/Modules/mediasession/MediaSession.h

```diff
@@ -67,6 +67,8 @@ class MediaSession : public RefCounted<MediaSession>, public ActiveDOMObject, pu
 
     void callActionHandler(const MediaSessionActionDetails&, DOMPromiseDeferred<void>&&);
 
+    auto& actionHandlers() { return m_actionHandlers; }
```

Comment:
should be marked as const. We don't want the callers to be able to modify the map.

Reply:
I took your advice below and wrote a function to iterate the map instead of this way of having the JSMediaSessionCustom code iterate the map.